tests/csm/rest/test_csm_users.py

```diff
@@ -236,7 +240,8 @@ def test_28936(self):
                 time.sleep(30)
         if not pod_up:
             assert pod_up, "Pod is not up so cannot proceed. Test Failed"
-        assert response1.status_code == HTTPStatus.SERVICE_UNAVAILABLE.value, "Account creation failed."
+        assert response1.status_code == HTTPStatus.SERVICE_UNAVAILABLE.value, (
```

Comment:
This can be HTTPStatus.SERVICE_UNAVAILABLE

Reply:
done